### DISCUSSION TITLE
help Calibrate SR10+ gyro  With ETX 2.9.2 on Radiomaster TX16S

### DISCUSSION THREAD

**Discussion by Fortuna599** [EdgeTX/edgetx#4568] [Q&A]
Is there a way to calibrate the gyro with the TX16s?

I'm trying to calibrate my new FrSky SR10+ receiver and I'm not succeeding, I've already updated the radio's firmware to ETX 2.9.2, internal module to version multi-stm-1.3.3.27_AETR.bin and the receiver is in version ARCHER-PLUS_SR10 +_1.0..5. using AP_SRx_Lua_v2.04 it stays stuck on the initial screen and does not start the stabilizer calibration. Can someone help me? 
Thanks

**Comment by pfeerick** [comment]
Please do not crosspost... someone will reply soon enough ;)

I'm running 1.07 on my SR10+, and am able to calibrate on both TX16S with MPM (Make sure you bind using Frsky X2 protocol, and probably a good idea to scan for sensors before trying to calibrate) and also X9D+2019 using ACCESS. 

I'm using the current 2.04 OTX Lua from the Frsky website. Make sure you download and install it again if you were using an old copy, as they seem to have updated it in place after the original release, as they realised they had the image paths wrong. There is also updated firmware for that RX... v1.0.10 as of a couple of days ago that may help. 

> Fixed the compatible issue of the receiver working with the Stab Lua scripts while running on some OPTX radio models

https://www.frsky-rc.com/archer-plus-sr10/

**Reply by Fortuna599** [reply]
Sorry, I made a request for help elsewhere by accident, I'm new here and also with the TX16s and FRsky. To install 2.04 OTX Lua, I'm putting it in the SDcard utilities folder and running the file, is this correct?
Is there any other way to install it?

**Reply by Fortuna599** [reply]
![WhatsApp Image 2024-01-26 at 08 18 41](https://github.com/EdgeTX/edgetx/assets/157658549/8e8bf83c-807f-4484-83e8-7537f47f5e8a)
After executing and pressing enter it does not leave this screen

![WhatsApp Image 2024-01-26 at 08 23 36](https://github.com/EdgeTX/edgetx/assets/157658549/0ed0c92b-ef7b-421a-b3a3-fbd772103f55)

**Reply by pfeerick** [reply]
Yes, in the SCRIPTS/TOOLS folder. It looks like you have it installed correctly as the images won't display if it's in the wrong folder. 

What are the lights on the SR10+ doing... you should have a green one rapidly blinking, and the yellow led should be off (and come one if you move the receiver sharply or tap it). If you have a blue one on, it's stuck in self-check mode, and you need to get it out of it... I think I had to follow the instructions for the 1.05 firmware where it said you had to user an older lua script to set up the stabilisation mode... which is hopefully what the fix referred to in the 1.0.10 update is in reference to. Although, I don't know if you can update a SR10+ in OTA when in ACCST mode since that's an ACCESS feature :-/ ... so maybe you'll have to do that also... which means you probably have to update it via the SPort... oh joy! These videos may help with that... 

https://youtu.be/0fNC2Jw1kxY
https://youtu.be/9tncdNDinFM?si=yIxLCqhCws0p9JMU&t=849

> Note2: To set up the Stabilization Mode on the Archer Plus SR10+ receiver using the OPTX transmitter, please roll back to 1.0.2 firmware version and download the OPTX script tool from the provided link below: https://www.frsky-rc.com/wp-content/uploads/Downloads/TOOL/R9S-SxR(lua)/FrSky_STAB_RX_2.03.zip

**Reply by Fortuna599** [reply]
I put an SD card without any files and the radio doesn't turn on anymore, when I press the power button it keeps beeping the buzzer, can you help me

**Reply by Fortuna599** [reply]
Yes, the green lights flash quickly and the yellow LED goes off and on if I move the receiver suddenly or touch it. Yes, I updated through SPort to 1.0.10 and tried to calibrate with version 2.03 and newer and it didn't work.
